docs/predicates.md

```diff
@@ -0,0 +1,17 @@
+# `ArborX::nearest`, `ArborX::intersects`
+Defined in header [`<ArborX_Predicates.hpp>`](https://github.com/arborx/ArborX/blob/master/src/details/ArborX_Predicates.hpp)
+
+```C++
+template <typename Geometry>
+KOKKOS_FUNCTION unspecified nearest(Geometry const& geometry, std::size_t k) noexcept; // (1)
+
+template <typename Geometry>
+KOKKOS_FUNCTION unspecified intersects(Geometry const& geometry) noexcept; // (2)
```

Comment:
I guess this is unfinished. But maybe don't use (1) and (2) for quite different things?